VideoLocker/src/main/java/org/edx/mobile/task/GetResponsesListTask.java

```diff
@@ -26,8 +26,12 @@ public ThreadComments call() throws Exception {
         try {
             if (threadId != null) {
                 if (isQuestionType) {
+                    if (shouldGetEndorsed) {
+                        return environment.getDiscussionAPI().getResponsesListForQuestion(threadId,
```

Comment:
Instead of passing hard-coded `true` and `false` to overloaded functions, we should change the signature of both functions and make them more verbose i.e. `getEndorsedResponsesForQuestion` & `getUnendorsedResponsesForQuestion`.

Bonus point: Maybe have comments on both as well explaining the reason for having separate functions.